flake.nix

```diff
@@ -7,7 +7,7 @@
   outputs =
     { self, nixpkgs, flake-utils, devshell }:
     {
-      overlay = import ./overlay.nix { };
```

Comment:
@Rizary here is the key issue

Reply:
Hmmm.. What behavior of this code that failed my build?